### PR TITLE
10.0 migrate users_ldap_populate

### PR DESCRIPTION
**Pull request by StephanRozendaal**
Changes include:
 - migrate the module to 10.0
 - add a README for the module

### CODE REVIEW

File: users_ldap_populate/README.rst [hunk 1]
```diff
@@ -0,0 +1,68 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+===================
+users_ldap_populate
```
Comment (lasley) [inline users_ldap_populate/README.rst]: It would be better to put a humanized name here, like `Users LDAP Populate`

**Review by lasley — APPROVED** [review]
One comment, but otherwise LGTM thanks @StephanRozendaal

File: users_ldap_populate/README.rst [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+
+#. Go to 'Settings' -> 'General Settings'
+#. In the LDAP category click on or add a LDAP configuration
+#. In the pop-up enter all the LDAP settings
+#. Click the populate button
+#. A new pop-up will notify you of how many users are added
+
+.. image:: https://odoo-community.org/website/image/ir.attachment/5784_f2813bd/datas
+   :alt: Try me on Runbot
+   :target: https://runbot.odoo-community.org/runbot/{server-tools}/{10.0}
```
Comment (dreispt) [inline users_ldap_populate/README.rst]: The link doesn't work, even if you remove the brackets in the URL.
You can use: `https://runbot.odoo-community.org/runbot/repo/149`

File: users_ldap_populate/README.rst [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+
+.. repo_id is available in https://github.com/OCA/maintainer-tools/blob/master/tools/repos_with_ids.txt
+.. branch is "8.0" for example
+
+Bug Tracker
+===========
+
+Bugs are tracked on `GitHub Issues
+<https://github.com/OCA/{server-tools}/issues>`_. In case of trouble, please
```
Comment (dreispt) [inline users_ldap_populate/README.rst]: Fix the link.

**Review by dreispt — CHANGES_REQUESTED** [review]
LGTM other than some minor comments

**Review by dreispt — APPROVED** [review]

**Pull request merged**